AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3845,63 +3847,63 @@ function Debug-AzStorageAccountEntraKerbAuth {
         }
         #
         # Port 445 check 
-        #
-        
+        #        
         if (!$filterIsPresent -or $Filter -match "CheckPort445Connectivity")
         {
+            Write-Host "Checking Port 445 Connectivity"
             try {
+                Write-Host "`tTesting ... " -NoNewline

```

Comment:
We're printing this:

```
Checking Port 445 Connectivity
   Testing ... Done
```

I think the "testing" is doing a bit of double duty with the "checking". Here's what I would suggest we do instead.

```
Checking Port 445 Connectivity... Done
```

So the code would be something like

```suggestion
            Write-Host "Checking Port 445 Connectivity... " -NoNewline
            try {
```

It also has the added advantage of being terser

Reply:
(and if you go with this you'd also have to remove the `` `r `` from the Write-TestingFailed)